src/libs/pool-data-provider/providers/static-pool-data-provider.tsx

```diff
@@ -82,7 +82,7 @@ export function StaticPoolDataProvider({
   );
 
   const activeData = isRPCActive && rpcData ? rpcData : cachedData;
-  if ((isRPCActive && rpcDataLoading) || (!isRPCActive && cachedDataLoading)) {
+  if ((isRPCActive && rpcDataLoading && !rpcData) || (!isRPCActive && cachedDataLoading)) {
```

Comment:
without this, there's a loading render flash whenever there's a new data polling